src/Http/Requests/Cart/CartRequest.php

```diff
@@ -16,7 +16,7 @@ public function getModelClass(): string
 
     public function authorize()
     {
-        return false;
+        return true;
```

Comment:
This is the default return value if you wanted to streamline this a bit.

Reply:
I discovered later that the custom base implementation isn't working quite right (not locating model parameter), so I've actually been pulling this method entirely in favor of  `$this->authorizeResource(...);` (though `Cart` remains non-standard since the user Cart magically always exists and isn't an actual route parameter)